aws/structure.go

```diff
@@ -740,6 +763,21 @@ func flattenElastiCacheParameters(list []*elasticache.Parameter) []map[string]in
 	return result
 }
 
+// Flattens an array of Parameters into a []map[string]interface{}
+func flattenNeptuneParameters(list []*neptune.Parameter) []map[string]interface{} {
+	result := make([]map[string]interface{}, 0, len(list))
+	for _, i := range list {
+		if i.ParameterValue != nil {
+			result = append(result, map[string]interface{}{
+				"name":         strings.ToLower(aws.StringValue(i.ParameterName)),
```

Comment:
The `strings.ToLower()` here seems extraneous. We can always add it back if for some reason its necessary.